CONTRIBUTING.md

```diff
@@ -431,6 +431,83 @@ After you have added new feature, to make sure it stays working, we recommend us
 
 [This](https://github.com/freeCodeCamp/chapter/contribute) is a good place to go if you are looking to get started.
 
+## Authorizing GraphGQL requests with `@Authorized` decorator
+
+### Overview or roles
+
+Chapter uses role based permission system, with specific permission assigned to one or multiple roles, on one or more different _levels/scopes_).
+
+* Instance role - per instance - every registered user has instance role.
+* Chapter role - per chapter - chapter role is created for user when explicitly joining chapter, or performing on event actions requiring chapter role. Ie. Rsvping to event. Elevated chapter role for one chapter, doesn't impact chapter roles for different chapters.
+* Event role - per event - chapter role is created for user when Rsvping to an event. Elevated event role (permissions) for a one event doesn't impact event roles for different events.
+
+_Note: in MVP Event roles are not used._
+
+### `@Authorized` decorator
+
+Decorator is available as an import from `type-graphql`.
+
+```ts
+import { Authorized } from 'type-graphql';
+```
+
+Adding it in resolver before specific request logic, wraps the request in the authorization logic. Only for authorized users logic of the request will be executed.
```

Comment:
```suggestion
Adding it in the resolver before specific request logic, will wrap the request in the authorization logic, which will execute only authorized users' logic of the request.
```

Reply:
The last part of the suggestion:
> which will execute only authorized users' logic of the request.

Changes the sense of the original sentence. The point is that when user isn't authorized, their request will not reach the actual code intended for handling that request (when user is authorized), but will be stopped earlier, at the authorization step.

Reply:
What do you think of this?

```suggestion
Structuring your authorization logic, by adding a resolver before the specific request, will wrap that request. With this, the request will execute that logic only for authorized users
```